65black/StackOverflow-lite
# bg(fix-status-codes): fix wrong status codes verification

#### What does this PR do?
When a user tries to access a password protected endpoint without a valid token, a wrong HTTP error code is thrown. This PR aims to fix that.

#### Description of Task to be completed?
Fixes errror in the following file:

`src/middlewares/verifyToken.js`

#### How should this be manually tested?
After cloning the repo and installing all dependencies with `npm install`

- Using Postman, add a new question without providing an access token in your header or using a wrong token.
- You should get the appropriate 401 error.

#### What are the relevant pivotal tracker stories?
#160143582

#### Screenshots (if appropriate)

![status](https://user-images.githubusercontent.com/36507972/44799171-3282f280-abab-11e8-8c8a-e789071a014c.PNG)

-------------------------------------------------------------------------------------------------------------------
![status2](https://user-images.githubusercontent.com/36507972/44799183-37e03d00-abab-11e8-994e-b122a16cbf31.PNG)
